### PR TITLE
chore: pull request template

### LINKED ISSUE #17: chore: pull request template
add a pull request template to the project for improved pull request description. 

for reference:
[Pull Request Template](https://learn.microsoft.com/en-us/azure/devops/repos/git/pull-request-templates?view=azure-devops)

### PR DESCRIPTION
**Pull request by kat-kan**
Fixes #17 
Please review and let me know what you think
I was thinking about adding also the checklist from CONTRIBUTING.md, but for now I kept it simple as discussed.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+-
+-
+
+## Notes
+
+<!-- Please provide additional notes here (if applicable). -->
+
+## Closing issues
+
```
Comment (olamide203) [inline .github/pull_request_template.md:19]: this part should be  a comment at the top something like
```md
<!-- If your PR fixes an open issue, use `Closes #101` to link your PR with the issue. #101 stands for the issue number you are fixing -->
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Put `Closes #XXXX` in your comment to auto-close the issue that your PR fixes (if applicable).
+
```
Comment (olamide203) [inline .github/pull_request_template.md:21]: this part should be  a comment at the top something like
```md
<!-- If your PR fixes an open issue, use `Closes #101` to link your PR with the issue. #101 stands for the issue number you are fixing -->
```

Comment (kat-kan) [reply .github/pull_request_template.md:21]: Hi, ofc I'll change, but do you mean at the top of Closing Issues part? Or do you want me to remove Closing Issues and add this comment at the top of the whole template? Thanks

Comment (olamide203) [reply .github/pull_request_template.md:21]: yeah, I think it's better to remove the closing issues section and have that comment at the top instead. Thanks.

Comment (kat-kan) [reply .github/pull_request_template.md:20]: @olamide203 just added the change :)

**Pull request merged**